great_expectations/datasource/fluent/data_asset/path/file_asset.py

```diff
@@ -91,7 +91,10 @@ def add_batch_definition_path(self, name: str, path: PathStr) -> BatchDefinition
             raise AmbiguousPathError(path=path)
         return self.add_batch_definition(
             name=name,
-            partitioner=FileNamePartitionerPath(regex=regex),
+            partitioner=FileNamePartitionerPath(
+                regex=regex,
+                param_names=(),
```

Comment:
question/nit: we don't really have to pass in `param_names` do we?

Reply:
we do, unfortunately, because [the FluentDatasource base model overrides pydantics default](https://github.com/great-expectations/great_expectations/blob/84cfd3d6b25aa5b9998e27bcb87e31b2b123a72a/great_expectations/datasource/fluent/fluent_base_model.py#L159) of `exclude_unset=False`. If we don't explicitly pass this, it gets dropped.